homeassistant/components/scrape/sensor.py

```diff
@@ -164,12 +201,14 @@ def _extract_value(self) -> Any:
 
     async def async_added_to_hass(self) -> None:
         """Ensure the data from the initial update is reflected in the state."""
-        await super().async_added_to_hass()
+        await ManualTriggerEntity.async_added_to_hass(self)
+        await CoordinatorEntity.async_added_to_hass(self)  # type: ignore[arg-type]
```

Comment:
Argument 1 to "async_added_to_hass" of "BaseCoordinatorEntity" has incompatible type "ScrapeSensor"; expected "BaseCoordinatorEntity[_BaseDataUpdateCoordinatorT]"  [arg-type]

Reply:
We had this discussion in a previous PR but I couldn't find it right now.
I change to super() and I can remove the ignore

Reply:
Yeah, but let's not write unclear code because of limitations in mypy.
I think this is the same problems as in this mypy issue: https://github.com/python/mypy/issues/15097

Reply:
As mentioned this is reverted to have clear code also ignoring the mypy issue for now.